lib/s.ml

```diff
@@ -122,9 +135,15 @@ module type BUILDER = sig
   val count : t -> int64
   (** [count t] return number of items in the store. *)
 
+  val root : t -> string
+  (** [root t] returns the root of the store. *)
+
   val df : t -> float Lwt.t
   (** [df t] returns the percentage of free space in the store. *)
 
+  val shell : t -> string list
+  (** [shell] optional value to be used as the default shell. *)
```

Comment:
Is this correct, to be consistent with the surrounding documentation, and because it is no longer an optional value?

```suggestion
  (** [shell t] returns the default shell. *)
```